HW2.js

```diff
@@ -0,0 +1,68 @@
+// Task #1 - Задача про обчислення різниці часу
+
+    function durationBetweenDates ( dateStart = '05 Oct 2020', dateEnd = '06 Oct 2023', dimension = 'days'){
+
+        let timeDiff =  Math.abs(new Date(dateEnd) - new Date(dateStart)); 
+
+        switch (dimension){
+            case ('days'): timeDiff /= (1000 * 3600 * 24); break;
+            case ('hours'): timeDiff /= (1000 * 3600); break;
+            case ('minutes'): timeDiff /= (1000 * 60); break;
+            case ('seconds'): timeDiff /= 1000; break;
+
+            default: return console.log('Wrong dimantion format!'); 
+        }
+
+        return timeDiff.toString()+' '+dimension;
```

Comment:
```suggestion
        return `${timeDiff.toString()} dimension`;
```
template literal теж саме, але зручніше